libraries/AC_Avoidance/AC_Avoidance_Logging.cpp

```diff
@@ -58,8 +59,9 @@ void AP_OADijkstra::Write_Visgraph_point(const uint8_t version, const uint8_t po
     };
     AP::logger().WriteBlock(&pkt, sizeof(pkt));
 }
+#endif
```

Comment:
```suggestion
#endif   // AP_OAPATHPLANNER_ENABLED
```

similarly anywhere else the excluded code is more than a few lines

Reply:
Done.